src/Fulma/Common.fs

```diff
@@ -7,15 +7,25 @@ open Fable.Helpers.React.Props
 module Color =
 
     module Classes =
-        let [<Literal>] IsBlack = "is-black"
-        let [<Literal>] IsDark = "is-dark"
-        let [<Literal>] IsLight = "is-light"
-        let [<Literal>] IsWhite = "is-white"
-        let [<Literal>] IsPrimary = "is-primary"
-        let [<Literal>] IsInfo = "is-info"
-        let [<Literal>] IsSuccess = "is-success"
-        let [<Literal>] IsWarning = "is-warning"
-        let [<Literal>] IsDanger = "is-danger"
+        let [<Literal>] IsBlack = "has-background-black"
```

Comment:
Careful using `is-dark` or `has-background-dark` doesn't have the same result.

```
<a class="button is-dark">Dark</a>
<a class="button has-background-dark">Black</a>
```

Give:

<img width="149" alt="capture d ecran 2018-04-29 a 08 23 09" src="https://user-images.githubusercontent.com/4760796/39403880-95a09924-4b86-11e8-8074-9606842d8c5c.png">

I am still not sure how we can correctly map the modifiers into Fulma. I was thinking of adding `Modifier` case to all the components because modifiers can be apply to any Bulma components.

Perhaps I should solve #97 first or at the same time we do #137 
If we want to explore add `Modifier` to all Fulma components it's not a trivial task then.

My idea is that a minimal Fulma components will be:

```fs
type GenericOption =
        | CustomClass of string
        | Props of IHTMLProp list
        | Modifiers of IModifiers list // Or something like that
```

```fs
type IModifiers =
    | BackgroudColor of IBackgroundColor
    | TextAlignement of ITextAlignement
    | ...
```

Reply:
Also because `has-background-*`, `is-*`, `has-text-*` use the same `$color` variable in Bulma we can use `IColor` for all three of them.

Reply:
So if I understand you correctly what needs to happen here is:
- create a general IModifier and add that to the GenericOption
- add 3 types of color: background, text, color
- make IColor independant
- create specific implementations for all three

Reply:
Create a general `IModifer` add it to all the components.

In the `IModifer` add a `background` and `text` case. The standard color case is already implement in the components who need it and need to stay where it is for now.

Create the `IColor` parser for `has-background` and `has-text`.

If you want to give it a try I am ok (and if I explain whell enough ^^) otherwise I will do it myself and ask feedback in a PR.